client/settings/app-settings.jsx

```diff
@@ -55,19 +51,28 @@ class AppForm extends React.Component {
     return (
       <form noValidate={true} onSubmit={onSubmit}>
         <SubmitOnEnter />
-        <VolumeSettings>
-          <StyledSlider
-            {...bindCustom('masterVolume')}
-            label='Master volume'
-            tabIndex={0}
-            min={0}
-            max={100}
-            step={1}
-            showTicks={false}
-            onChange={this.onMasterVolumeChange}
-          />
-          <TestSoundButton label={testSoundLabel} onClick={onTestSoundClick} />
-        </VolumeSettings>
+        <FormContainer>
+          <div>
+            <StyledSlider
```

Comment:
You can just wrap this component and the test sound button component in a `VolumeSettingsContainer` component that already existed to get them aligned as they were previously. This would also eliminate the need to fiddle with margins of the button that probably would've needed to be done with the new layout.

But also make sure to leave the outer `div` in place as that one wraps the first column and will be needed when we add more items to it.

Reply:
So like?

```
        <FormContainer>
          <div>
            <VolumeSettings>
              <StyledSlider
                {...bindCustom('masterVolume')}
                label='Master volume'
                tabIndex={0}
                min={0}
                max={100}
                step={1}
                showTicks={false}
                onChange={this.onMasterVolumeChange}
              />
              <TestSoundButton label={testSoundLabel} onClick={onTestSoundClick} />
            </VolumeSettings>
          </div>
          <div>
            <CheckBox
              {...bindCheckable('runAppAtSystemStart ')}
              label='Run ShieldBattery on system startup'
              inputProps={{ tabIndex: 0 }}
            />
          </div>
        </FormContainer>
        ```

Reply:
Yep, that ought to do it. I think ^^